mlflow/pytorch/__init__.py

```diff
@@ -287,6 +288,93 @@ def gen_data():
     )
 
 
+@experimental
+def log_state_dict(state_dict, artifact_path, **kwargs):
+    """
+    Log a state dict as an MLflow artifact for the current run.
+
+    :param state_dict: PyTorch state dict to be saved.
+    :param artifact_path: Run-relative artifact path.
+    :param kwargs: kwargs to pass to ``torch.save`` method.
+
+    .. code-block:: python
+        :caption: Example
+
+        import torch
```

Comment:
I think the example here is an overkill and difficult to understand.

```python
with mlflow.start_run():
    state_dict = model.state_dict()
    mlflow.pytorch.log_state_dict(state_dict, artifact_path="state_dict")
```

Reply:
I used the same example as in log/save model. I have updated the code given above.